packages/stats/gbstats/models/results.py

```diff
@@ -67,3 +67,9 @@ class ExperimentMetricAnalysisResult:
 class ExperimentMetricAnalysis:
     metric: str
     analyses: List[ExperimentMetricAnalysisResult]
+
+
+@dataclass
+class MultipleExperimentMetricAnalysis:
```

Comment:
This class doesn't look like it's being used anywhere

Reply:
Now it is used properly. The stats engine isn't aware it needs to return a json serializable object anymore, and this is happened in the string of text we execute in `stats.ts`.